src/dtscalibration/io/sensortran.py

```diff
@@ -39,16 +39,17 @@ def read_sensortran_files(
         keyword-arguments are passed to DataStore initialization
 
     Returns:
-    -------
+    --------
     DataStore
         The newly created datastore.
     """
     filepathlist_dts = sorted(Path(directory).glob("*BinaryRawDTS.dat"))
 
     # Make sure that the list of files contains any files
-    assert (
-        len(filepathlist_dts) >= 1
-    ), "No RawDTS measurement files found " "in provided directory: \n" + str(directory)
+    assert len(filepathlist_dts) >= 1, (
```

Comment:
Another one (I think I added some of the assertions back in the day...). Probably best to open an issue about it.